dspace-spring-rest/src/main/java/org/dspace/app/rest/utils/DiscoverQueryBuilder.java

```diff
@@ -183,7 +183,7 @@ private DiscoverQuery buildCommonDiscoverQuery(Context context, DiscoveryConfigu
         if (StringUtils.isNotBlank(query)) {
             //Note that these quotes are needed incase we try to query OR for example.
             //If the quotes aren't present, it'll crash.
```

Comment:
@Raf-atmire : Have we tested this change with boolean OR queries?  The comment here specifically says the quotes are needed in the `setQuery()` call, or else boolean OR won't work, etc. If the quotes are definitely not needed, we should also remove the comment.

Reply:
@tdonohue : I think I typed my explanation wrong. OR queries will work if we're talking about query= dc.date.issued:2007-01-08 OR dc.date.issued:2007-01-09 for example.
A query that doesn't work is query=OR I was told that this query not working properly was a deliberate design and it will be brought up in a coming DSpace meeting.

Reply:
@Raf-atmire : Ok, then I'd suggest removing the two line comment here as part of this PR.  The comment no longer describes the code.

Reply:
@tdonohue : That's a good point, I've removed the comments in a new commit. Thanks!